### DISCUSSION TITLE
这个仓库的作用是什么，这个也许是个超级有用的框架，但是我不了解

### DISCUSSION THREAD

**Discussion by 75ACOL** [QilinPTA/Qilin#6] [Q&A]
早上好，我不太理解这个指针分析指的是什么，可以举一个例子么。我不理解这个框架能做什么

**Comment by DongjieHe** [comment]
Hi, here is the definition of pointer analysis [https://en.wikipedia.org/wiki/Pointer_analysis](https://en.wikipedia.org/wiki/Pointer_analysis) with illustrating examples.
